tests/sparseml/pytorch/sparsification/pruning/test_mask_creator.py

```diff
@@ -161,3 +162,35 @@ def test_four_block_mask_creator_matches_block(tensors, sparsity_val):
     for mask_1, mask_2 in zip(masks_1, masks_2):
         assert mask_1.shape == mask_2.shape
         assert torch.all(mask_1 == mask_2)
+
```

Comment:
do we have any way here of testing that the created masks comply to the N:M spec?